coala_quickstart/generation/Bears.py

```diff
@@ -418,3 +429,52 @@ def prompt_to_activate(bear, printer):
         return False
     else:
         return prompt_to_activate(bear, printer)
+
+
+def ask_to_select_capabilties(all_capabilities,
+                              default_capabilities,
+                              console_printer):
+    """
+    Asks the users to select capabilties out of all_capabilities.
+    """
+    all_capabilities = sorted(all_capabilities)
+    indentation = " " * 4
+    PROMPT_QUESTION = ("What would you like the bears to detect or fix? "
+                       "Please select some bear capabilities using "
+                       "their numbers:")
+    console_printer.print(PROMPT_QUESTION, color="yellow")
+
+    for idx, cap in enumerate(all_capabilities):
+        color = 'cyan'
+        if cap in default_capabilities:
+            color = 'green'
+        console_printer.print(
+            indentation + str(idx + 1) + '.' + cap, color=color)
+    console_printer.print(
+        indentation + str(idx + 2) + '.' + "Select all default capabilities " +
```

Comment:
also mention that they can just press enter to chose this option (because your try-except below allows that).